modules/nw-egress-ips-about.adoc

```diff
@@ -149,6 +149,67 @@ On Azure, the following capacity limits exist for IP address assignment:
+
+If the egress IP is not within the subnet of the primary network interface, then the selection of another network interface for egress traffic might occur if they are present on a node. 
+
+You can determine which other network interfaces might support egress IPs by inspecting the `k8s.ovn.org/host-cidrs` Kubernetes node annotation. This annotation contains the addresses and subnet mask found for the primary network interface. It also contains additional network interface addresses and subnet mask information. These addresses and subnet masks are assigned to network interfaces that use the link:https://networklessons.com/cisco/ccna-200-301/longest-prefix-match-routing[longest prefix match routing] mechanism to determine which network interface supports the egress IP. 
+
+[NOTE]
+====
+OVN-Kubernetes provides a mechanism to control and direct outbound network traffic from specific namespaces and pods. This ensures that it exits the cluster through a particular network interface and with a specific egress IP address.
+====
+
+[discrete]
+[id="nw-egress-ips-multi-nic-requirements_{context}"]
+=== Requirements for assigning an egress IP to a network interface that is not the primary network interface
+
+For users who want an egress IP and traffic to be routed over a particular interface that is not the primary network interface, the following conditions must be met:
+
+* {product-title} is installed on a bare metal cluster. This feature is disabled within cloud or hypervisor environments.
+
+* Your {product-title} pods are not configured as host-networked.
+
+* A standard Linux network interface is required and must be one of bond, vlan, or ethernet types. If a network interface is removed or if the IP address and subnet mask which allows the egress IP to be hosted on the interface is removed, then the egress IP is reconfigured. Consequently, it could be assigned to another node and interface.
+
+* The Egress IP must be IPv4. IPv6 is currently unsupported.
+
+* IP forwarding must be enabled for the network interface. To enable IP forwarding, you can set `ipForwarding: Global` in your `config.yaml` file, for example:
++
+[source,yaml]
+----
+# ...
+ipForwarding: Global
+# ...
+----
+
+[discrete]
+[id="nw-egress-ips-multi-nic-limitations_{context}"]
+=== Limitations for assigning an egress IP to a network interface that is not the primary network interface
+
+The following limitations apply when assigning an egress IP to a network interface that is not the primary network interface:
+
+* Only network interfaces of types of bond, ethernet, or vlan are supported.
```

Comment:
One more concern is that myabe we don't need to explicitly state the supported network interfaces here.  As we only tested the ethernet type.   Better to remove this line. 
cc @martinkennelly

Reply:
I have tested them manually yesterday and its ok but I guess since QE hasnt, lets remove it.

Reply:
@martinkennelly Do we want to remove it from here too? 

* A standard Linux network interface is required and must be one of bond, vlan, or ethernet types. If a network interface is removed or if the IP address and subnet mask which allows the egress IP to be hosted on the interface is removed, then the egress IP is reconfigured. Consequently, it could be assigned to another node and interface.

Reply:
> A standard Linux network interface is required and must be one of bond, vlan, or ethernet types. 

Can remove this sentence.